src/components/analyses/AnalysesNavigation.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @author sriram
+ *
+ * A component that acts as a Navigation bar to the analyses listing view.
+ * It contains primary ways to filter the analyses view.
+ */
+
+import React from "react";
+import ids from "./ids";
+import { build, formatMessage, getMessage, withI18N } from "@cyverse-de/ui-lib";
+import {
+    Divider,
+    Hidden,
+    IconButton,
+    Link,
+    makeStyles,
+    TextField,
+    Toolbar,
+    Tooltip,
+    Typography,
+} from "@material-ui/core";
+import { injectIntl } from "react-intl";
+import messages from "./messages";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import { getAppTypeFilters } from "../apps/AppNavigation";
+import ownershipFilter from "./model/ownershipFilter";
+import FilterListIcon from "@material-ui/icons/FilterList";
+import ClearIcon from "@material-ui/icons/Clear";
+
+const useStyles = makeStyles((theme) => ({
+    menuButton: {
+        color: theme.palette.primary.contrastText,
+    },
+    divider: {
+        flexGrow: 1,
+    },
+    filter: {
+        [theme.breakpoints.down("xs")]: {
+            width: 130,
+        },
+        [theme.breakpoints.up("sm")]: {
+            width: 150,
+        },
+        margin: theme.spacing(1.5),
+    },
+    batchFilter: {
+        [theme.breakpoints.down("sm")]: {
+            textOverflow: "ellipsis",
+            whiteSpace: "nowrap",
+            overflow: "hidden",
+            maxWidth: 75,
+        },
+    },
+}));
+
+function getOwnershipFilters() {
+    return Object.keys(ownershipFilter).map((filter) => {
+        return {
+            name: ownershipFilter[filter],
+        };
+    });
+}
```

Comment:
I think you could also do: 

```suggestion
function getOwnershipFilters() {
    return Object.values(ownershipFilter).map((filter) => {
        return {
            name: filter,
        };
    });
}
```